docs/man/policy/kas-registry/key/_index.md

```diff
@@ -8,8 +8,10 @@ command:
     - k
   flags:
     - name: json
-      description: output single command in JSON (overrides configured output format)
+      description: Output the result of a subcommand in JSON format (overrides configured output format). This is an inherited flag.
```

Comment:
Note: I think this is unnecessary if this command is a child of `policy`?

Reply:
@jakedoublev I was wondering that. Should we just remove the json flag from any command we added under policy?

Reply:
I removed this